src/web/EmployerFavouritesWeb/Startup.ConfigureServices.cs

```diff
@@ -45,6 +45,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMediatR(typeof(Startup).Assembly);
 
+            services.AddHealthChecks()
+                .AddCheck<FavouritesApiHealthCheck>("favourites-api-check");
+
```

Comment:
Might be worth adding redis check also

Reply:
We don't use Redis for caching in Employer Favourites currently. Or did you mean checking the logging Redis?

Reply:
Ah yes the basket!